conda-recipe-openblas/meta.yaml

```diff
@@ -10,7 +10,7 @@ requirements:
     - numpy
     - openblas >=0.3.0
     - libflang  # [win]
-    - libgfortran # [not win]
+    - libgfortran # [not win]    
```

Comment:
Why extra spaces?

Reply:
dunno. Editing on windows gives me that stuff sometimes.... Will remove them.